build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/AbstractCheckOrUpdateContributorsInReleaseNote.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gradlebuild.buildutils.tasks
+
+import com.google.gson.Gson
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.work.DisableCachingByDefault
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+data class GitHubPullRequestSearchResult(val items: List<GitHubPullRequest>)
+data class GitHubUser(val login: String, val name: String?)
+data class GitHubPullRequestMilestone(val title: String)
+data class GitHubPullRequest(
+    val number: Int,
+    val user: GitHubUser,
+    val milestone: GitHubPullRequestMilestone?,
+)
+
+@DisableCachingByDefault(because = "Depends on GitHub API")
+abstract class AbstractCheckOrUpdateContributorsInReleaseNote : DefaultTask() {
+    @get: Internal
+    val releaseNote: RegularFileProperty = project.objects.fileProperty()
```

Comment:
```suggestion
    val releaseNotes: RegularFileProperty = project.objects.fileProperty()
```

Let's use `releaseNotes` consistently.

Reply:
Done.